packages/components/organisms/f-takeawaypay-activation/src/components/ActivationLoggedIn.vue

```diff
@@ -4,7 +4,7 @@
 
         <h1
             :class="$style['c-takeawaypayActivation-title']">
-            {{ $t('messages.titleActivationAvailable') }}
+            {{ $t('messages.titleActivationAvailable1') }}{{ consumerGivenName }}. {{ $t('messages.titleActivationAvailable2') }}
```

Comment:
Feel like this could be a computed property, getting a little long for markup

Reply:
@matthewhardern  Move to the computed property 👍 f